Documentation/Signoffs/LaunchingAndFiring.md

```diff
@@ -0,0 +1,84 @@
+| 6. | Preloading after firing | The Launching and Firing subsystem shall reset the preload condition after each round is fired within 0.79 seconds. | Team Specification | 
+
+
+## Buildable Schematic
+### Electrical Schematic 
+![alt text](https://github.com/JTJones73/Capstone2024-Team2/blob/jlburke42-signoff-Launching-and-Firing/Documentation/Images/CapstoneDrawing2.png)
+
+As seen in the schematic above, the Adafruit relay will be powered off the 3.3V pin (PIN1) of the Raspberry Pi 5. While the GPIO pins are limited to 50mA, this pin can support 3.3V up to 500mA. The Adafruit 3191 relay only draws 100mA while operating. A signal will be sent from GPIO 6 (PIN31) which shall switch allow the relay to switch from connecting "COMM" and "Normally Connected" to connecting "COMM" to " Normally Closed" allowing the motor to be powered from the 12V bus from the Housing subsystem. This motor shall draw 9A upon startup and when reaching it's stall point and will be wired with 16 AWG wire from the Housing subsystem.
+
+## Analysis 
+
+**Constraint Solution 1:**
+The Launching and Firing subsystem shall use 16 AWG wire for the subsystems electrical connections. This wire has an ampacity rating of 10 amps over a distance of 12 inches. This wire size was selected by considering the largest ampacity draw that exists in the subsystem. For this the team decided to use the start/stall current (9.8 amps) of the 12V DC motor used in this subsystem to apply mechanical power to the firing mechanism. Using a table [4] that seems to simplify and directly quotes the NEC 310.15(B)(16), the team decided to use 14 AWG wire[5] than can handle over 15 Amps, leaving plenty of capacity in the wire, this wire will be sufficient to use for this application. 
+
+**Constraint Solution 2:**
+To allow the firing mechanism to operate without human interaction, a signal shall be generated from the Aiming subsystem when that subsystem has finished the aiming processes that will be used as a relay signal to operate the firing mechanism autonomously. The relay[3] will disconnect the 12-volt supply from the DC motor when the signal is not active, or low, and will connect the 12-volt supply to the DC motor when the signal is activated, or high, allowing the motor to operate. The relay will be connected to pin 1 of the Raspberry Pi 5 from the aiming subsystem and will pull 100mA at max (while operating) which is within the Raspberry Pi 5's ampacity limits for that pin. The relay, the Adafruit Power Relay FeatherWing G5LE-G is rated for 10 Amp resistive loads at 12 Volts DC. This is over the motor's stall ampacity and it also useful to consider that the team will be operating the motor at much ampacity than this at all times however this relay can handle 10 amps at 12 volts regardless.
```

Comment:
Look at the relay again. What DC current can it handle? Note that relays can handle higher ac current than dc current.

Reply:
Hey Professor, I was able to add a proximity sensor to the design and told the ME team this was to be incorporated into the design based on the placement of where they needed it to be, but electrically and for our purposes, this document has been updated. Also, I made notes of when geared torque was in question, what side of the gearing I was on. For the relay, the image below seems to point me to a switching current of 10A DC, and rated for 10A up to 35VDC, if you have any insight on how it may differ please let me know! Thanks
![image](https://github.com/user-attachments/assets/876843f5-b6db-46f6-b3bc-9f9e128ab4b3)